test_runner/regress/test_pg_regress.py

```diff
@@ -258,8 +262,21 @@ def test_isolation(
     with capsys.disabled():
         pg_bin.run(pg_isolation_regress_command, env=env_vars, cwd=runpath)
 
-    # This fails with a mismatch on `pg_multixact/offsets/0000`
-    # post_checks(env, test_output_dir, DBNAME, endpoint)
+    # Flush and check all that we wrote is compactable
+    post_checks(
+        env,
+        test_output_dir,
+        DBNAME,
+        endpoint,
+        extra_ignores=[
+            # These files may be absent in pgdata after running `pg_isolation_regress`
+            "pg_multixact/offsets/0000",
```

Comment:
You could now enable the post_checks call earlier in the file (see comment 'This fails with a mismatch on `pg_multixact/offsets/0000`')